server/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
Why are you doing this?  Loopback already has the server instance started elseware...

Reply:
I added 'require(dotenv)' so that it could read the environment variable from the .env file and eslint was complaining that 'use strict' wasn't on there so i added it and it resolved the issue.